storage/mroonga/vendor/groonga/lib/ii.c

```diff
@@ -4808,7 +4808,6 @@ exit :
   return ctx->rc;
 }
 
-PRAGMA_REENABLE_CHECK_STACK_FRAME
```

Comment:
So the next `PRAGMA_REENABLE_CHECK_STACK_FRAME` was extraneous before your patch?
Odd.

Reply:
removing this is allowing the disable to continue though `grn_ii_delete_one`.

Reply:
I needed the following function having stack frame disabled.